dictionary/Dictionary.js

```diff
@@ -45,6 +45,14 @@ export default class Dictionary {
 	 *         evaluated using the provided parameters.
 	 */
 	get(key, parameters = {}) {}
+
+	/**
+	 * Tests whether the localization phrase exists at dictionary
+	 *
+	 * @param {string} key The key identifying the localization phrase.
+	 * @return {boolean | null} {@code true} if the the key (phrase) exists. {@code null} if key exists, but has others children keys.
```

Comment:
1. Please use closure compiler-compatible type annotations: `{?boolean}`
2. "Returns {@code null} if..." - please use complete sentences in the documentation. (the verb can be implied by the annotation tag for the first sentence).
3. s/the the/the/
4. s/if key/if the key/
5. "...but has child keys."
6. Please respect the 80 character line limit, tabs are considered to be 4-space wide.
7. Returning `null` for mid-nodes is questionable: should a condition such as `if (dictionary.has('foo')) { ... }` pass or fail? What is the expected use case for this feature?